NEWS

```diff
@@ -1,3 +1,148 @@
+
+A new command line utility, projinfo, has been added to query information about
+a geodetic object of the database, import and export geodetic objects from/into
+WKT and PROJ strings, and display coordinate operations available between two
+CRSs.
+
+ UPDATES
+ -------
+
+ o Removed projects.h as a public interface (#835)
+
+ o Deprecated the proj_api.h interface. The header file is still available
+   but will be removed with the next major version release of PROJ. It is
+   now required to define ACCEPT_USE_OF_DEPRECATED_PROJ_API_H before the
+   interface can be used (#836)
+
+ o Removed support for the nmake build system (#838)
+
+ o Removed support for the proj_def.dat defaults file (#201)
+
+ o C++11 required for building PROJ (#1203)
+
+ o Added build dependency on SQLite 3.7 (#1175)
+
+ o Added projinfo command line application (#1189)
+
+ o Added many functions to proj.h for handling ISO19111 functionality (#1175)
+
+ o Added C++ API exposing ISO19111 functionality (#1175)
+
+ o Updated cs2cs to use late-binding features (#1182)
+
+ o Removed the nad2bin application. Now available in the proj-datumgrid
+   git repository (#1236)
+
+ o Removed support for Chebyshev polynomials in proj (#1226)
+
+ o Removed proj_geocentric_latitude from proj.h API (#1170)
+
+ o Changed behaviour of proj: Now only allow initialization of
+   projections (#1162)
+
+ o Changed behaviour of tmerc: Now default to the Extended Transverse
+   Mercator algorithm (etmerc). Old implementation available by adding
+   +approx (#404)
+
+ o Chaged behaviour: Default ellipsoid now set to GRS80 (was WGS84) (#1210)
+
+ o Allow multiple directories in PROJ_LIB environment variable (#1281)
+
+ o Added Lambert Conic Conformal (2SP Michigan) projection (#1142)
+
+ o Added Bertin1953 projection (#1133)
+
+ o Added Tobler-Mercator projection (#1153)
+
+ o Added Molodensky-Badekas transform (#1160)
+
+ o Added push and pop coordinate operations (#1250)
+
+ o Removed +t_obs parameter from helmert and deformation (#1264)
+
+ o Added +dt parameter to deformation as replacement for
+   removed +t_obs (#1264)
+
+ BUG FIXES
+ ---------
+
+ o Read +towgs84 values correctly on locales not using dot as comma separator (#1136)
+
+ o Fixed file offset for reading of shift values in NTv1 files (#1144)
+
+ o Avoid problems with PTHREAD_MUTEX_RECURSIVE when using CMake (#1158)
+
+ o Avoid raising errors when setting ellipsoid flattening to zero (#1191)
+
+ o Fixed lower square calculations in rHealpix projection (#1206)
+
+ o Allow Molodensky transform parameters to be zero (#1194)
+
+ o Fixed wrong parameter in ITRF2000 init file (#1240)
+
+ o Fixed use of grid paths including spaces (#1152)
+
+ THANKS TO
+ ------------
+
+ Version 6.0.0 is made possible by the following contributors:
+
+Phil Elson
+Mike Taves
+Mateusz Łoskot
+Markus Neteler
+Jürgen Fischer
+Howard Butler
+Charles Karney
+Alan D.
+Aaron Puchert
+Karoline Skaar
+Marco Bernasocchi
+Mike Toews
```

Comment:
I'm listed twice. Remove "Mike Toews" and keep just "Mike Taves". Perhaps this list should also be sorted.

Reply:
Why do you go by two names? I am just basing this of what you guys report yourself in your git commits. Also, the list is sorted:

https://github.com/OSGeo/proj.4/blob/a813ad65cfb819bfa295dfbacdf577b258377c93/HOWTO-RELEASE#L85

Reply:
You can fix:

> You may need to edit the list slightly since not all authors has properly configured their names in git.

by adding a `.mailmap`.

Reply:
The list seems not sorted or prioritised (me before @hobu !?) ;P

Reply:
> The spelling was changed last year to be more pronounceable. One name should appear in future commit logs.

Ah, alright. Congrats on the new, more pronounceable name!

> The list seems not sorted or prioritised (me before @hobu !?) ;P

@mloskot I can assure you that you are equally important according to my list :-)

```
>git shortlog -sn 5.2.0..HEAD | tac
     1  magau
     1  Phil Elson
     1  Mike Taves
     1  Mateusz Łoskot
     1  Markus Neteler
     1  Jürgen Fischer
     1  Howard Butler
     1  Charles Karney
     1  Alan D. Snow
     1  Aaron Puchert
     2  Mike Toews
     2  Marco Bernasocchi
     2  Karoline Skaar
     3  Ben Boeckel
     5  Philippe Rivière
     5  Ivan Veselov
     6  Elliott Sales de Andrade
   106  Kristian Evers
   262  Even Rouault
```